### PR TITLE
Multiplayer helptext updates for RTD

### PR DESCRIPTION
**Pull request by jwrober**
Associated with output from pyciv. Part of https://github.com/longturn/freeciv21/issues/1452 and https://github.com/longturn/freeciv21/issues/1453.

### CODE REVIEW

File: data/multiplayer/README.multiplayer [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
-1. Changes compared to the classic ruleset
-   ---------------------------------------
+Changes Compared to the Classic Ruleset
+=======================================
 
 The differences can be extracted by
 
-#> diff -ur data/classic data/multiplayer
+$ diff -ur data/classic data/multiplayer
```
Comment (lmoureaux) [inline data/multiplayer/README.multiplayer:17]: Does this get rendered as fixed-width text?

File: data/multiplayer/README.multiplayer [hunk 2]
```diff
@@ -93,63 +92,68 @@ As for the remaining great wonders:
...
 
 To play a game with the multiplayer rules, start the server with:
-       ./fcser -r data/multiplayer.serv
+
+$ ./freevic21-server -r data/multiplayer.serv
+
 or a single player game with
-       ./freeciv-gtk3 -r data/multiplayer.serv
+
+$ ./freeciv21-client -r data/multiplayer.serv
```
Comment (lmoureaux) [inline data/multiplayer/README.multiplayer:146]: Same question here

**Review by lmoureaux — APPROVED** [review]

File: data/multiplayer/README.multiplayer (hunk 1 above)
Comment (jwrober) [reply data/multiplayer/README.multiplayer:17]: Nope, neither does the `#>`. that is why I went with the `$` make it look more like a command prompt.

File: data/multiplayer/README.multiplayer (hunk 2 above)
Comment (jwrober) [reply data/multiplayer/README.multiplayer:146]: same answer. I didn't want to add a bunch of rst type code to the plain text file so tried to add simple stuff that would at least look like plain text but still render ok in RTD.

**Pull request merged**